### PR TITLE
Update Registration to Include Under 16

### PR DESCRIPTION
**Pull request by wgranger**
Describe your changes.
Adds text changes to the registration form when a user is under 16.
https://docs.google.com/presentation/d/1t4u2L9TPZ1j7qq5yfzw7MvyTocDGY78Af8SPdC97x9A/edit#slide=id.p

Screen Shot 2019-04-01 at 10 35 42 AM

Normally, this component would be rewritten into ES6 with tests. However, we're trying to make these changes quickly, and a rewrite will have to wait.

# Required Manual Testing

- [x] Does the non-logged in home page render correctly?
- [x] Does the logged in home page render correctly?
- [x] Does the projects page render correctly?
- [x] Can you load project home pages?
- [x] Can you load the classification page?
- [x] Can you submit a classification?
- [x] Does talk load correctly?
- [x] Can you post a talk comment?

# Review Checklist

- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

**Comment by eatyourgreens** [comment]
Should be staged by Jenkins at https://pr-5306.pfe-preview.zooniverse.org

**Comment by beckyrother** [comment]
Small design note: is there a reason 'Required' is floating sort of in the middle? Can it be right aligned to the input boxes?

**Comment by beckyrother** [comment]
Also does 'Parent/Guardian email' change to 'your email' if the box is un-ticked?

**Comment by wgranger** [comment]
@beckyrother The current site has a floating "required," but I can make the change in this PR.

Yes, unticking the box will revert to default.

File: app/partials/register-form.cjsx [hunk 1]
```diff
@@ -60,6 +72,11 @@ module.exports = createReactClass
     {badNameChars, nameConflict, passwordTooShort, passwordsDontMatch, emailConflict} = @state
 
     <form method="POST" onSubmit={@handleSubmit}>
```
Comment (eatyourgreens) [inline app/partials/register-form.cjsx]: I don't see any changes to `handleSubmit` in this PR. Is Panoptes recording that this account is under 16?

Comment (wgranger) [reply app/partials/register-form.cjsx]: I don't think so. I believe the only changes are to the frontend content, but we should get a backend confirmation on this before merge.

Comment (eatyourgreens) [reply app/partials/register-form.cjsx]: It seems odd to ask this on the registration form, but not actually record which user accounts ticked it.

**Comment by wgranger** [comment]
This branch should be good to go.

**Comment by mrniaboc** [comment]
LGTM

**Comment by mrniaboc** [comment]
What has to be done to get this merged?

**Comment by eatyourgreens** [comment]
Press the merge button?

**Comment by mrniaboc** [comment]
Hahaha I saw that not all the things had green ticks against them, so I was unsure.

**Comment by eatyourgreens** [comment]
I think Jenkins might have crashed. All the PRs for PFE are showing as waiting on Jenkins.

**Pull request merged**